### DISCUSSION TITLE
Cara menggunakan googleAuthR

### DISCUSSION THREAD

**Discussion by rafiff23** [teamalgoritma/community#608] [Q&A] [answered]
Saya ingin membuat shiny dengan login page dimana login page tersebut menggunakan email dari google. kebanyakan artikel dan contoh shiny yang sudah di buat di internet adalah versi lama sedangkan googleauthR sekarang sudah di versi 2.0.0 Apakah dari tim algo bisa membuat app.R dummy untuk menjelaskan?

**Comment by Yusraf002** [comment]
Dear @rafiff23  ,
Terima kasih atas pertanyaannya pak, tim kami akan segera coba bantu atau carikan referensi ya pak

**Comment by Yusraf002** [comment, marked as answer]
Dear @rafiff23,

Berikut salah satu link yang dapat Bapak gunakan untuk mempelajari pembuatan login page dan Authentication Management for Shiny Applications

- [Authentication Management for Shiny Applications](https://datastorm-open.github.io/shinymanager/)

Demikian yang dapat saya sampaikan, semoga membantu.
